tests/integration/buildcmd/build_integ_base.py

```diff
@@ -713,19 +717,23 @@ class BuildIntegProvidedBase(BuildIntegBase):
     EXPECTED_FILES_PROJECT_MANIFEST = {"__init__.py", "main.py", "requests", "requirements.txt"}
 
     FUNCTION_LOGICAL_ID = "Function"
-    code_uri = "Provided"
 
-    def _test_with_Makefile(self, runtime, use_container, manifest, architecture=None):
+    def _test_with_Makefile(
```

Comment:
found this test helper function, so I just added the ability to customize the code uri and whether to build in source